ci/code_checks.sh

```diff
@@ -65,1541 +65,1236 @@ fi
+        --ignore_errors pandas.tseries.offsets.Week.n GL08\
+        --ignore_errors pandas.tseries.offsets.Week.name SA01\
+        --ignore_errors pandas.tseries.offsets.Week.nanos GL08\
+        --ignore_errors pandas.tseries.offsets.Week.normalize GL08\
+        --ignore_errors pandas.tseries.offsets.Week.rule_code GL08\
+        --ignore_errors pandas.tseries.offsets.Week.weekday GL08\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth PR02,SA01\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.copy SA01\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.freqstr SA01\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.is_on_offset GL08\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.kwds SA01\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.n GL08\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.name SA01\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.nanos GL08\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.normalize GL08\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.rule_code GL08\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.week GL08\
+        --ignore_errors pandas.tseries.offsets.WeekOfMonth.weekday GL08\
+        --ignore_errors pandas.tseries.offsets.YearBegin PR02\
+        --ignore_errors pandas.tseries.offsets.YearBegin.copy SA01\
+        --ignore_errors pandas.tseries.offsets.YearBegin.freqstr SA01\
+        --ignore_errors pandas.tseries.offsets.YearBegin.is_on_offset GL08\
+        --ignore_errors pandas.tseries.offsets.YearBegin.kwds SA01\
+        --ignore_errors pandas.tseries.offsets.YearBegin.month GL08\
+        --ignore_errors pandas.tseries.offsets.YearBegin.n GL08\
+        --ignore_errors pandas.tseries.offsets.YearBegin.name SA01\
+        --ignore_errors pandas.tseries.offsets.YearBegin.nanos GL08\
+        --ignore_errors pandas.tseries.offsets.YearBegin.normalize GL08\
+        --ignore_errors pandas.tseries.offsets.YearBegin.rule_code GL08\
+        --ignore_errors pandas.tseries.offsets.YearEnd PR02\
+        --ignore_errors pandas.tseries.offsets.YearEnd.copy SA01\
+        --ignore_errors pandas.tseries.offsets.YearEnd.freqstr SA01\
+        --ignore_errors pandas.tseries.offsets.YearEnd.is_on_offset GL08\
+        --ignore_errors pandas.tseries.offsets.YearEnd.kwds SA01\
+        --ignore_errors pandas.tseries.offsets.YearEnd.month GL08\
+        --ignore_errors pandas.tseries.offsets.YearEnd.n GL08\
+        --ignore_errors pandas.tseries.offsets.YearEnd.name SA01\
+        --ignore_errors pandas.tseries.offsets.YearEnd.nanos GL08\
+        --ignore_errors pandas.tseries.offsets.YearEnd.normalize GL08\
+        --ignore_errors pandas.tseries.offsets.YearEnd.rule_code GL08\
+        --ignore_errors pandas.unique PR07\
+        --ignore_errors pandas.util.hash_array PR07,SA01\
+        --ignore_errors pandas.util.hash_pandas_object PR07,SA01
+    )
```

Comment:
I'd like to suggest keeping a comment like this on line 1294 
`# There should be no backslash in the final line, please keep this comment in the last ignored function`
This way, as folks work through clearing these up we can remember this line shouldn't end in \

Reply:
Good point. Done